eth2/simulation/src/simulation.rs

```diff
@@ -15,37 +17,205 @@ where
 }
 
 impl<T: EthSpec> Simulation<T> {
+    pub fn new() -> Self {
+        Self {
+            store: Store::new(),
+        }
+    }
+
     /// Add a new execution environment, return EE index
-    fn create_execution_environment(a: args::CreateExecutionEnvironment) -> EEIndex {
-        unimplemented!();
+    pub fn create_execution_environment(
+        &mut self,
+        a: args::CreateExecutionEnvironment,
+    ) -> Result<EEIndex> {
+        // Create EE struct from args
+        let vl_wasm_code =
```

Comment:
Is the `vl_` prefix a convention from the lighthouse repository?

I think I'd prefer to see just `wasm_code`, and use the shorthand assignment in the `ExecutionEnvironment` initialization.

Reply:
I agree, I'll just use `wasm_code`.